.lintstagedrc

```diff
@@ -1,6 +1,5 @@
 {
   "*.js": "prettier --write",
   "*.ts": "prettier --write", 
-  "*.md": "prettier --write",
-  "*.sol": "prettier --write"
```

Comment:
I had to remove sol here b.c. the sol specific prettier settings are only setup in the core package, not across the monorepo. We can probably configure it higher up if we want. Tbh I'm not sure how this if/how this ever worked